.github/workflows/docusaurus.yml

```diff
@@ -0,0 +1,28 @@
+name: docusaurus
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build Docusaurus
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Set up Yarn
+        working-directory: ./doc
+        run: corepack enable && corepack prepare yarn@4.3.1 --activate
```

Comment:
Does `corepack enable` work? Now that we check in the `.yarn/releases` directory, I think we don't need to specify the exact yarn version.

Reply:
Done

Reply:
I didn't realize we can remove it entirely, cool!